troubadix/standalone_plugins/file_extensions.py

```diff
@@ -0,0 +1,72 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+# SPDX-FileCopyrightText: 2024 Greenbone AG
+
+import sys
+from argparse import ArgumentParser, Namespace
+from pathlib import Path
+from typing import Iterable, List
+
+
+def directory_type(string: str) -> Path:
+    directory_path = Path(string)
+    if directory_path.exists() and not directory_path.is_dir():
+        raise ValueError(f"{string} is not a directory.")
+    return directory_path
+
+
+def parse_args(args: Iterable[str]) -> Namespace:
+    parser = ArgumentParser(
+        description="Check for changed oid",
```

Comment:
Seems this also requires an update as this doesn't match what the check is actually doing.

Reply:
implemented in a2d497e6c79b262ccdbb0ce889e143ae0f714968